src/components/GenerateLabelFile.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import shortId from 'shortid';
+import downloadFile from 'js-file-download';
+
+function buildLabelRow() {
+    const id = shortId.generate();
```

Comment:
Is this protecting us from generating IDs that are redundant with labels already created?

Reply:
shortId is just a wrapper around UUID, but rather than only using base16 (0-9A-F) as the characters, it uses [a-zA-Z0-9-] as its characters so it can fit the same UUID into a smaller set of characters. 

This will make for a shorter URL for the QR code and an easier ID to type in if it needs to be typed in from somewhere.